test/model/TestExecutionStateTest.php

```diff
@@ -0,0 +1,231 @@
+<?php
+
+namespace oat\taoMediaManager\test\model;
+
+
+
+use oat\tao\test\TaoPhpUnitTestRunner;
+use oat\taoMediaManager\model\MediaManagerManagement;
```

Comment:
There is no dependencies between MediaManager and this extension, we should not call namespace from extension mediamanager, could you check these usages and clean that